src/Components/DetailsPage.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import Header from './DetailsPageHeader';
+
+const DetailsPage = () => {
+  const location = useLocation();
+  const data = location.state;
+
+  return (
+    <>
+      <Header />
+      <div className="details-container">
+        <h2>{data.name}</h2>
+        <div className="details-content">
+          <div className="image-container">
+            <img src={data.flag} alt="Country flag" />
+          </div>
+          <div className="details">
+            {
+            data.nativeName
+              ? (
+                <p>
+                  Native Name:
+                  <span className="lighter">{data.nativeName}</span>
+                </p>
+              ) : ''
+          }
+            {
+            data.population
+              ? (
+                <p>
+                  Population:
+                  <span className="lighter">{data.population}</span>
+                </p>
+              ) : ''
+          }
+            {
+            data.region
+              ? (
+                <p>
```

Comment:
- [ ] it would be better to stick to the given design as required and try to style the information as  in the design with a rectangular shape in order to match the design 
- Screenshot of the requirements 
<img width="529" alt="Screenshot 2023-02-14 at 15 43 22" src="https://user-images.githubusercontent.com/61790587/218771023-dea2e0ee-43b3-4e36-b3d9-809154d73581.png">


- Yours 
![Screenshot 2023-02-14 at 15 42 23](https://user-images.githubusercontent.com/61790587/218770739-723291bd-000b-4547-a35b-ddb1e6c8eb39.png)

- Design 
<img width="200" alt="Screenshot 2023-02-14 at 15 39 52" src="https://user-images.githubusercontent.com/61790587/218770589-2a084e93-e36c-4d30-96db-a84d23b80ce9.png">

Reply:
Thanks for your observation, changes have been implemented 🙏